api/server/router/build/build_routes.go

```diff
@@ -67,6 +69,10 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 		options.Isolation = i
 	}
 
+	if runtime.GOOS != "windows" && options.SecurityOpt != nil {
+		return nil, fmt.Errorf("the daemon on this platform does not support --security-opt to build")
```

Comment:
Hmm. Not sure about that. This is consistent with https://github.com/docker/docker/blob/master/api/server/router/build/build_routes.go#L65. And has to be picked up from the options (via `r.Form["securityopt"] a few lines higher).